.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        versions:
+          - python: 2.7
+            toxenv: py27
+            os: ubuntu-latest
+          - python: 3.4
+            toxenv: py34
+            os: ubuntu-18.04
```

Comment:
This one fails the build with:

```
pkg_resources.VersionConflict: (Pygments 2.3.1 (/opt/hostedtoolcache/Python/3.4.10/x64/lib/python3.4/site-packages), Requirement.parse('Pygments>=2.5.1'))
```

Since it's EOL, we might want to drop it.